frontend/src/components/common/YesNoRadioButtons/yesNoRadioButtons.styles.ts

```diff
@@ -5,6 +5,7 @@ export const StyledSpan = styled.span`
   font-size: 16px;
 `;
 
+// Unused
```

Comment:
Note: I added this comment. I kept the styling in case you wanted to use it. If this is unneeded, you can delete it.